soc/wch/ch32v/soc.yml

```diff
@@ -7,9 +7,7 @@ family:
   - name: qingke-v2a
     socs:
     - name: ch32v003
-  - name: ch32v00x
```

Comment:
The ch32v006 is not a qingke-v2a but a v2c

Reply:
If it's not clear i'm quite a bit annoyed CH32V006 was merged, because it messes up this PR and means it requires ANOTHER round of reviews and hours of update for me, and obviously I have made errors on adding ch32v006 to it, but I'm really not inclined in spending more time on it while everyone has been dragging their feets about this PR.

Reply:
And you know, the fact that it's probably going to need another round of reorganization as discussed there https://github.com/zephyrproject-rtos/zephyr/pull/87345#discussion_r2061206095 and so it's basically doubly wasted time.

Reply:
I hear you, and I'm sorry for your wasted time. When I reviewed it, this PR breaks both the ch32v003 and ch32v00x, so I can't LGTM it as is.

Could I suggest breaking this up into separate PRs covering separate concerns? For example, the CH32V208 PLL, flash, and startup changes are independent of the rename.

Reply:
I am working on fixing it so it doesnt break them and they are organized as they are supposed to be in this version.

Reply:
It should be good now.